CastleNathria/Locales/ruRU.lua

```diff
@@ -28,75 +28,75 @@ L = BigWigs:NewBossLocale("Lady Inerva Darkvein", "ruRU")
-	-- L.custom_on_repeating_dark_recital_desc = "Repeating Dark Recital say messages with icons {rt1}, {rt2} to find your partner while dancing."
+	L.macabre_start_emote = "Занимайте свои места и приготовьтесь к пляске смерти!" -- [RAID_BOSS_EMOTE] Take your places for the Danse Macabre!#Dance Controller#4#false"
+	L.custom_on_repeating_dark_recital = "Повторение Тёмного бала"
+	L.custom_on_repeating_dark_recital_desc = "Спам сообщений в /сказать с метками {rt1}, {rt2} чтобы найти своего партнёра для танца во время Тёмного бала."
 
-	-- L.custom_off_select_boss_order = "Mark Boss Kill Order"
-	-- L.custom_off_select_boss_order_desc = "Mark the order the raid will kill the bosses in with cross {rt7}. Requires raid leader or assist to mark."
+	L.custom_off_select_boss_order = "Маркировка порядка убийства боссов"
+	L.custom_off_select_boss_order_desc = "Устанавливать метку крест {rt7} в соответствии с порядком убийства боссов. Требуется быть помощником или лидером рейда."
 	L.custom_off_select_boss_order_value1 = "Никлаус -> Фрида -> Ставрос"
 	L.custom_off_select_boss_order_value2 = "Фрида -> Никлаус -> Ставрос"
 	L.custom_off_select_boss_order_value3 = "Ставрос -> Никлаус -> Фрида"
 	L.custom_off_select_boss_order_value4 = "Никлаус -> Ставрос -> Фрида"
 	L.custom_off_select_boss_order_value5 = "Фрида -> Ставрос -> Никлаус"
 	L.custom_off_select_boss_order_value6 = "Ставрос -> Фрида -> Никлаус"
 
-	--L.dance_assist = "Dance Assist"
-	--L.dance_assist_desc = "Show directional warnings for the dancing stage."
-	--L.dance_assist_up = "|T450907:0:0:0:0:64:64:4:60:4:60|t Dance Forward |T450907:0:0:0:0:64:64:4:60:4:60|t"
-	--L.dance_assist_right = "|T450908:0:0:0:0:64:64:4:60:4:60|t Dance Right |T450908:0:0:0:0:64:64:4:60:4:60|t"
-	--L.dance_assist_down = "|T450905:0:0:0:0:64:64:4:60:4:60|t Dance Down |T450905:0:0:0:0:64:64:4:60:4:60|t"
-	--L.dance_assist_left = "|T450906:0:0:0:0:64:64:4:60:4:60|t Dance Left |T450906:0:0:0:0:64:64:4:60:4:60|t"
+	L.dance_assist = "Помощник танцев"
+	L.dance_assist_desc = "Показ предупреждений о направлении во время фазы танцев."
+	L.dance_assist_up = "|T450907:0:0:0:0:64:64:4:60:4:60|t Двигайся вперед |T450907:0:0:0:0:64:64:4:60:4:60|t"
+	L.dance_assist_right = "|T450908:0:0:0:0:64:64:4:60:4:60|t Двигайся направо |T450908:0:0:0:0:64:64:4:60:4:60|t"
+	L.dance_assist_down = "|T450905:0:0:0:0:64:64:4:60:4:60|t Двигайся вниз |T450905:0:0:0:0:64:64:4:60:4:60|t"
+	L.dance_assist_left = "|T450906:0:0:0:0:64:64:4:60:4:60|t Двигайся влево |T450906:0:0:0:0:64:64:4:60:4:60|t"
 	-- These need to match the in-game boss yells
-	--L.dance_yell_up = "Forward" -- Prance Forward!
-	--L.dance_yell_right = "right" -- Shimmy right!
-	--L.dance_yell_down = "down" -- Boogie down!
-	--L.dance_yell_left = "left" -- Sashay left!
+	L.dance_yell_up = "вперед" -- Prance Forward!
+	L.dance_yell_right = "вправо" -- Shimmy right!
+	L.dance_yell_down = "назад" -- Boogie down!
+	L.dance_yell_left = "влево" -- Sashay left!
```

Comment:
Could you add the full yell in the comments here for future reference?

Reply:
fixed